frontend/assets/stylesheets/components/_packages.scss

```diff
@@ -109,6 +115,68 @@ $_package-promo__actions_height: 117px;
     }
 }
 
+//NEW
+.package-promo--clean {
+
+    @include mq($until: phablet) {
+        @include package-promo-narrow();
+        border: none;
+        background: $white;
+    }
+    @include mq($from: phablet) {
+        @include package-promo-wide();
+
+        border: none;
+        background: $white;
+        .package-promo__actions {
+            vertical-align: middle;
+
+        }
+    }
+
+    .package-promo__content {
+        display: flex;
```

Comment:
The "mobile-first" convention is to define styles for smaller devices first and override with media queries for larger devices, e.g. here a mobile style outside the media query, and then `@include mq($from: phablet)` with the phablet & up specific styles